### ISSUE TITLE
image w3m not working 

### ISSUE THREAD

**Issue by ward6slick** [dylanaraps/neofetch#1279]
## Description

Just re-installed for Debian 10. on urxvt w3m image previews work fine in ranger, but i cant get them to work in neofetch.

### Neofetch version
Neofetch version: 6.0.0
## Screenshot
the ascii shows up instead of w3m and wallpaper specified.
## Config file
https://pastebin.com/raw/QSeJmVFs
## Verbose log
when i run `neofetch -vv` i get this 
https://pastebin.com/raw/cjCvC9PL
then i installed `imagemagick` and `neofetch -vv` is outputting this
https://pastebin.com/raw/nAjVmLka
and `neofetch` is outputting as if im running `clear`

**Comment by dylanaraps** [comment]
The second verbose log isn't complete. Try and run it again.

**Comment by ward6slick** [comment]
> The second verbose log isn't complete. Try and run it again.

This is what I get when I run `neofetch -vv 2> neofetchlog` https://pastebin.com/raw/hzfVBzdV
does it have to do with this section?

+ image_backend
+ [[ w3m != \o\f\f ]]
+ type -p convert
+ case "${image_backend:-off}" in
+ get_image_source
+ case "$image_source" in
+ get_wallpaper

**Comment by ward6slick** [comment]
I fixed it. I had to switch how internal label was listed.
it was listed as `*internalBorder: 8` and i switched it to `URxvt.internalBorder: 8`
i saw this issue: https://github.com/dylanaraps/neofetch/issues/1174 it would be cool to allow '`*internalBorder' but its no biggie.
also i dont know if this stems from the same issue or completely different, but the default gap is 3 but now it looks like the default gap (which ive always used 3 and have scrots saved and compared them) when it is have set to 1. just letting u know man.
anyways, love neofetch and pywal, thank you.
